src/SemanticDataType/IntegerNumber.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Tiime\EN16931\SemanticDataType;
+
+class IntegerNumber implements Number
+{
+    public function __construct(private readonly int $value)
```

Comment:
```suggestion
    public function __construct(private readonly int $value)
```
Triggers a deprecation warning when passing a float in PHP 8.1 & 8.2
We must enable [strict typing](https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.strict) here for it to throw a TypeError.

Note : we should plan on adding strict typing on all our files.

Reply:
w/o strict typing : https://3v4l.org/8oS1N

w/ strict typing : https://3v4l.org/keSkN

Reply:
I've fixed that